ArduCopter/heli.cpp

```diff
@@ -211,10 +218,18 @@ void Copter::heli_update_autorotation()
     }
 }
 
-// heli_set_autorotation - set the autorotation flag throughout libraries
+// heli_set_autorotation - set the autorotation f`lag throughout libraries
 void Copter::heli_set_autorotation(bool autorotation)
 {
     motors->set_in_autorotation(autorotation);
 }
+
+#else
+//ensuring that the autorotation and bailout flags are set false in non-SITL boards
+void Copter::heli_update_autorotation()
+{
+    heli_flags.in_autorotation = false;
+    motors->set_enable_bailout(false);
+}
```

Comment:
Yes, this looks good.  Leaves no doubt on the status of bailout.  and leaves us the framework to include the manual bailout option when we are ready